### PR TITLE
[Bug] Fix Action-sheet shiftLeft binding

### LINKED ISSUE #494: [BUG] shiftLeft on Action Sheet no longer works.
**Describe the bug**
The `shiftLeft` binding on the `go-action-sheet` no longer works as intended.

**To Reproduce**
1. In version 1.5.2+
2. Create an action-sheet with the `shiftLeft` binding set to `true`
3. Start app and open action sheet

**Expected behavior**
When `shiftLeft` is applied, it should move the `go-action-sheet` to the left and the arrow to the right end of the `go-action-sheet`.

**Screenshots**
Screen Shot 2020-02-24 at 10 37 50 AM

### PR DESCRIPTION
**Pull request by grahamhency**
## PR Checklist
Please check if your PR fulfills the following requirements:
- [x] The commit message follows our guidelines: https://github.com/mobi/goponents/blob/master/CONTRIBUTING.md
~~Tests for the changes have been added (for bug fixes / features)~~
- [x] Docs have been added / updated (for bug fixes / features)

## PR Type
What kind of change does this PR introduce?
- [x] Bugfix
- [ ] Feature
- [x] Code style update (formatting, local variables)
- [x] Refactoring (no functional changes, no api changes)
- [ ] Build related changes
- [ ] CI related changes
- [x] Documentation content changes
- [ ] Other... Please describe:

## What is the current behavior?
Currently the `shiftLeft` binding on the `go-action-sheet` component does not position the action sheet content appropriately.
Closes #494 

## What is the new behavior?
Makes the `shiftLeft` binding work as intended

## Does this PR introduce a breaking change?
- [ ] Yes
- [x] No

### CODE REVIEW

File: projects/go-style-guide/src/app/features/ui-kit/components/action-sheet-docs/components/action-sheet-overview/action-sheet-overview.component.html [hunk 1]
```diff
@@ -135,7 +135,6 @@ <h2 class="go-heading-2">Placement</h2>
         <div class="go-column--100 go-column go-column--no-padding">
           <p class="go-body-copy">
             A simple example to demo different placements defined by <code class="code-block--inline">placement</code>.
-            For example, to define a right placement action sheet the code would be:
             <code [highlight]="placementGenericHtml"></code>
           </p>
```
Comment (jaredami) [inline projects/go-style-guide/src/app/features/ui-kit/components/action-sheet-docs/components/action-sheet-overview/action-sheet-overview.component.html]: Do we want to add a `<h4 class="go-heading-4 go-heading--underlined">Code</h4>` here?

File: projects/go-style-guide/src/app/app.component.html [hunk 2]
```diff
@@ -1,5 +1,23 @@
...
+            panelContent="GitHub"
+            externalLink="https://github.com/mobi/goponents">
+          </go-panel>
+          <go-panel
+            panelContent="NPM"
+            externalLink="https://www.npmjs.com/package/@tangoe/goponents">
+          </go-panel>
+        </ng-container>
+      </go-action-sheet>
+    </ng-container>
```
Comment (jaredami) [inline projects/go-style-guide/src/app/app.component.html]: 🥇

File: projects/go-style-guide/src/app/features/ui-kit/components/action-sheet-docs/components/action-sheet-overview/action-sheet-overview.component.html (hunk 1 above)
Comment (grahamhency) [reply projects/go-style-guide/src/app/features/ui-kit/components/action-sheet-docs/components/action-sheet-overview/action-sheet-overview.component.html]: Sure, I went ahead and added that in.

**Comment by grahamhency** [comment]
@jaredami I fixed the Travis issue, can I get another look here?

**Pull request merged**